docs/source/basic/installation.rst

```diff
@@ -0,0 +1,103 @@
+
+**Install alpaka:**
+
+.. code-block:: bash
+
+   # alpaka requires a boost installation, see alpaka documentation
+   git clone --depth 1 --branch 0.8.0 https://github.com/alpaka-group/alpaka.git
+   mkdir alpaka/build
+   cd alpaka/build
+   cmake ..
+   cmake --install .
+
+**Install vikunja:**
+
+.. code-block:: bash
+
+  git clone https://github.com/alpaka-group/vikunja.git
+  mkdir vikunja/build 
+  cd vikunja/build  
+  cmake ..
+  cmake --build .
+  cmake --install .
+
+Build Tests and Examples
+------------------------
+
+Enable and run the tests:
+
+.. code-block:: bash
+
+   # start in the vikunja project folder
+   mkdir build && cd build
+   cmake .. -DBUILD_TESTING=ON
+   cmake --build .
+   ctest
+
+Enable and run an example:
+
+.. code-block:: bash
+
+   # start in the vikunja project folder
+   mkdir build && cd build
+   cmake .. -DVIKUNJA_BUILD_EXAMPLES=ON
+   cmake --build .
+   ./example/transform/example_transform
+
+
+Use vikunja in a CMake project via ``find_package``
+---------------------------------------------------
+
+Once vikunja and alpaka are successfully installed, you can use them in your project via the cmake function ``find_packge``.
+
+.. code-block:: cmake
+
+   cmake_minimum_required(VERSION 3.18)
+   project(vikunjaProject)
+
+   find_package(vikunja REQUIRED)
+
+   alpaka_add_executable(${CMAKE_PROJECT_NAME} main.cpp)
+   target_link_libraries(${CMAKE_PROJECT_NAME} PRIVATE vikunja::vikunja)
+
+At the cmake configuration time of your project, you need to enable at least on alpaka accelerator to execute a vikunja function on a processor. The accelerators are enabled via the cmake argument ``-DALPAKA_ACC_<...>_ENABLE=ON``. All existing accelerators can be found `here <https://alpaka.readthedocs.io/en/latest/advanced/cmake.html>`_.
+
+.. code-block:: bash
+
+   # start in the folder of the root CMakeLists.txt
+   mkdir build && cd build
+   # enable serial CPU and CUDA GPU accelerator
+   cmake .. -DALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE=ON -DALPAKA_ACC_GPU_CUDA_ENABLE=ON
+   cmake --build .
+
+By default ``find_package(vikunja)`` runs ``find_package(alpaka)``, if the ``alpaka::alpaka`` target is not already defined.
+
+Use vikunja in a CMake project via ``add_subdirectory``
+-------------------------------------------------------
+
+Vikunja also provides cmake integration via ``add_subdirectory``. The `add_subdirectory <https://cmake.org/cmake/help/latest/command/add_subdirectory.html>`_ approach does not require vikunja or alpaka to be installed and allows for easy deployment of a custom vikunja version together with your project.
```

Comment:
```suggestion
Vikunja also provides CMake integration via ``add_subdirectory``. The `add_subdirectory <https://cmake.org/cmake/help/latest/command/add_subdirectory.html>`_ approach does not require vikunja or alpaka to be installed and allows for easy deployment of a custom vikunja version together with your project.
```

And the link should probably not point to cmake-latest but to the version you require for vikunja.